lib/fix_db_schema_conflicts/tasks/db.rake

```diff
@@ -8,7 +10,7 @@ namespace :db do
         "#{Rails.root}/db/schema.rb"
       end
       rubocop_yml = File.expand_path('../../../../.rubocop_schema.yml', __FILE__)
-      `bundle exec rubocop --auto-correct --config #{rubocop_yml} #{filename}`
+      `bundle exec rubocop --auto-correct --config #{rubocop_yml} #{filename.shellescape}`
```

Comment:
Curious - I tried this from the Rails console and got an undefined method, but after checking out your branch and testing `rake db:migrate` locally it works just fine. Weird.